README.md

```diff
@@ -29,8 +28,7 @@ Users of Ubuntu 64Bit wanting to run the 32bit variants might install all depend
 Users of ArchLinux (or Manjaro Linux) in 64 bit might have problems running TVTower (32 bit). If you run these distros in a VM or with an Intel GPU the graphics context is bugged and leads to an segfault. This does _not_ happen with "NG"-builds of TVTower (issue still under research).
   
 ##### Windows/Mac
-Should run without further requirements
-
+No requirements to run.
```

Comment:
Doesn't this translate to something like "there is no need to run the game"? Sounds a bit odd.